apps/root/src/pages/home/components/token-icon-multichain/index.tsx

```diff
@@ -26,14 +26,27 @@ const StyledNetworkLogosContainer = styled.div`
   bottom: -4px;
 `;
 
-export const TokenNetworksTooltipTitle = ({ balanceTokens }: { balanceTokens: BalanceToken[] }) => {
+export const TokenNetworksTooltipTitle = ({
+  balanceTokens,
+  isTokenBreakdown,
+}: {
+  balanceTokens: BalanceToken[];
+  isTokenBreakdown?: boolean;
+}) => {
   const intl = useIntl();
 
   return (
     <ContainerBox flexDirection="column" gap={1}>
       {balanceTokens.map((chainData, index) => (
         <ContainerBox key={index} gap={1} alignItems="center" justifyContent="start">
-          <TokenIcon token={toToken({ logoURI: getGhTokenListLogoUrl(chainData.token.chainId, 'logo') })} size={3.5} />
+          <TokenIcon
+            token={
+              isTokenBreakdown
```

Comment:
what is this for?

Reply:
We used to show network icons (for multi-chain token breakdown). In this case, we would need to show every reward token icon instead of it's network icon